src/chapters/06/main.tex

```diff
@@ -470,22 +470,22 @@ \section{Wider context}\label{sec:game_theory_wider_context}
 process: Moran processes or replicator dynamics. A good text book to read on
 these topics is~\cite{maschler2013game}.
 
-John Nash whose life was portrayed in the 2001 movie ``a beautiful mind'' (which
-is an adaptation of~\cite{nasar2011beautiful}) won the Noble prize
-for\cite{nash1950equilibrium} in which he proved that a Nash equilibrium
+John Nash whose life was portrayed in the 2001 movie ``A Beautiful Mind'' (which
+is an adaptation of~\cite{nasar2011beautiful}) won the Nobel Prize
+for~\cite{nash1950equilibrium} in which he proved that a Nash equilibrium
 always exists. However, in~\cite{nash1951non} John Nash gives an
-application of Game Theory to a specific version of Poker.
+application of game theory to a specific version of poker.
 
 Another application of the concept of Nash equilibrium
 is~\cite{deo2011centralized} where the authors identify worst case
 scenarios for ambulance diversion: a practice where an emergency room will
 declare itself too full to accept new patients. When they are multiple emergency
 units serving a same population strategic behaviour becomes relevant. The
 authors of this paper identify the effect of this decentralised decision making
-and also propose an approach that is socially optimal: similarly to the Taxi
+and also propose an approach that is socially optimal: similarly to the taxi
 problem considered in this chapter.
 
-A specific area of a lot of research in Game theory is the study of cooperative
+A specific area of a lot of research in game theory is the study of cooperative % TODO This last paragraph is so vague it is meaningless. Vince can you reword/add something?
```

Comment:
I can't really without going in to a huge amount of irrelevant detail. I disagree that it is much more vague or meaningless than other similar paragraphs.

```suggestion
A specific area of a lot of research in game theory is the study of cooperative
```